Source/Scene/Expression.js

```diff
@@ -1,26 +1,75 @@
 /*global define*/
 define([
+       '../Core/Color',
+       '../Core/defined',
        '../Core/defineProperties',
-       '../ThirdParty/jsep',
+       '../ThirdParty/jsep'
     ], function(
+        Color,
+        defined,
         defineProperties,
         jsep) {
     "use strict";
 
+    var LITERAL = 0;
+    var UNARY = 1;
+    var BINARY = 2;
+
     /**
      * DOC_TBA
      */
     function Expression(styleEngine, expression) {
         this._styleEngine = styleEngine;
 
-        console.log(jsep(expression));
+        var ast = jsep(expression);
+        this._runtimeAst = undefined;
+
+        console.log(ast);
+        this._runtimeAst = createRuntimeAst(ast);
+        console.log(this._runtimeAst);
     }
 
     defineProperties(Expression.prototype, {
     });
 
+    function Node(type, value) {
+        this._type = type;
+        this._value = value;
+    }
+
+    function createRuntimeAst(ast) {
+        var node;
+
+        if (ast.type === 'Literal') {
+            node = new Node(LITERAL, ast.value);
+        } else if (ast.type === 'CallExpression') {
+            var call = ast.callee.name;
```

Comment:
It can be a separate PR, but there are a lot of syntax errors we should catch here.